public/app/features/inspector/InspectDataOptions.tsx

```diff
@@ -0,0 +1,148 @@
+import React, { FC } from 'react';
```

Comment:
This is just moved from `public/app/features/inspector/InspectDataTab.tsx`, so **InspectDataTab** component is smaller.